src/main/java/com/brandwatch/internship/fetchandstoretweets/services/utility/TweetsToMentions.java

```diff
@@ -0,0 +1,28 @@
+package com.brandwatch.internship.fetchandstoretweets.services.utility;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Mention;
+import org.springframework.social.twitter.api.Tweet;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TweetsToMentions {
```

Comment:
This class creates mentions from tweets for a query. The methods will fit well in the `MentionService`.